crates/ruff_python_ast/src/visitor.rs

```diff
@@ -666,14 +667,35 @@ pub fn walk_type_param<'a, V: Visitor<'a> + ?Sized>(visitor: &mut V, type_param:
     match type_param {
         TypeParam::TypeVar(TypeParamTypeVar {
             bound,
+            default_value,
             name: _,
             range: _,
         }) => {
             if let Some(expr) = bound {
                 visitor.visit_expr(expr);
             }
+            if let Some(expr) = default_value {
+                visitor.visit_expr(expr);
+            }
```

Comment:
I might've missed this in the PEP if it's mentioned but I just want to confirm if this is the correct order for this visitor. This should be in the evaluation order. I'm asking because for parameters, defaults are evaluated before the annotations.

Reply:
They are evaluated lazily, similar to bounds, so there isn't really an evaluation order.